src/item.cpp

```diff
@@ -1655,6 +1655,8 @@ stacking_info item::stacks_with( const item &rhs, bool check_components, bool co
               same_type && ( count_by_charges() || charges == rhs.charges ) );
     bits.set( tname::segments::FAVORITE_PRE, is_favorite == rhs.is_favorite );
     bits.set( tname::segments::FAVORITE_POST, is_favorite == rhs.is_favorite );
+    bits.set( tname::segments::CUSTOM_ITEM_PREFIX );
+    bits.set( tname::segments::CUSTOM_ITEM_SUFFIX );
```

Comment:
This should check if `this` and `rhs` have the same set of flags with prefixes/suffixes, because otherwise they should not stack.